qcelemental/physical_constants/__init__.py

```diff
@@ -2,13 +2,4 @@
 Units handlers
 """
 
-# Ensure pint exists
-from importlib.util import find_spec
-spec = find_spec('pint')
-if spec is None:
-    raise ModuleNotFoundError(
-        """Python module pint not found. Solve by installing it: `conda install pint -c conda-forge` or `pip install pint`"""
-    )
-del spec, find_spec
```

Comment:
I think there is some misunderstanding. I don't think we should edit this file at all and leave as it was before this PR. This is the optimal solution and the small duplication of code isn't worth the additional delayed importing hassles.